docs/topics/features.adoc

```diff
@@ -3,7 +3,7 @@
 // * docs/getting-started-guide/master.adoc
 
 [id="features_{context}"]
-= {ProductShortName} Features
+= {ProductShortName} features
 
 The {ProductName} ({ProductShortName}) provides a number of capabilities to assist with planning and executing migration projects.
```

Comment:
Can you make this less verbose? Also, I thought you were going to get rid of 'executing' globally.